tests/functional/z_cluster/test_ceph_default_values_check.py

```diff
@@ -153,6 +150,67 @@ def test_validate_ceph_config_values_in_rook_config_override(self):
             f"The cluster's Ceph values are:{config_data}"
         )
 
+    def _validate_config_post_420(self, ocs_version) -> None:
+        """
+        Validate ceph config values for OCS versions >= 4.20 using CephCluster resource.
+
+        Args:
+            ocs_version: OCS version for determining expected config values.
+
+        Returns:
+            None
+        """
+        ceph_cluster_obj = OCP(
+            kind=constants.CEPH_CLUSTER,
+            namespace=config.ENV_DATA["cluster_namespace"],
+            resource_name=constants.CEPH_CLUSTER_NAME,
+        )
+        ceph_cluster_data = ceph_cluster_obj.get()
+        ceph_config = ceph_cluster_data["spec"]["cephConfig"]
+
+        log.info(
+            "Validating that the Ceph values, configured in CephCluster spec.cephConfig, "
+            "match the ones stored in ocs-ci"
+        )
+        log.info(f"CephConfig data: {ceph_config}")
+
+        # Get expected values based on version
+        if ocs_version >= version.VERSION_4_20:
```

Comment:
We need to change to the line `if ocs_version == version.VERSION_4_20:`

Reply:
This is a placeholder for future version, we will adjust it *if* numbers will change in next release